playbook/app/pb_kits/playbook/pb_phone_number_input/_phone_number_input.scss

```diff
@@ -51,24 +98,28 @@ $flag-min-resolution: 192dpi;
     }
   }
 
-  .iti--allow-dropdown .iti__flag-container:hover .iti__selected-flag {
+  .iti--allow-dropdown .iti__country-container:hover .iti__selected-country {
     background-color: $focus_input_light;
   }
 
-  .iti__flag-container:hover + .text_input {
+  .iti__country-container:hover + .text_input {
     background-color: $focus_input_light;
   }
 
   .iti__flag {
-    background-image: url("https://unpkg.com/playbook-ui@14.2.0-alpha.fixphonenumberinputflag3631/dist/assets/flags.png");
+    background-image: url("https://unpkg.com/intl-tel-input@24.6.0/build/img/flags.png");
```

Comment:
I believe using the intl-tel-input would be as stable as using an unpkg link for a playbook alpha. Let me know if you think I should use a playbook alpha, though.